agent/packaging/ubuntu_xenial/kontena-agent/etc/kontena-agent.env

```diff
@@ -0,0 +1,11 @@
+# Set URL of your kontena server
+#KONTENA_URI=wss://localhost:8443
+
+# Set kontena token
+#KONTENA_TOKEN=
+
+# kontena peer interface (overlay network interface)
+KONTENA_PEER_INTERFACE=eth1
+
+# kontena version
+KONTENA_VERSION=latest
```

Comment:
We should pin the version because otherwise these packages cannot be used properly with configuration management systems (chef/ansible etc).